### PR TITLE
Tweak CacheDir docs

### PR DESCRIPTION
**Pull request by mwichmann**
Formatting, some wordsmithing. This is a doc-only change.

## Contributor Checklist:

* [ ] I have created a new test or updated the unit tests to cover the new/changed functionality.
* [X] I have updated `CHANGES.txt` (and read the `README.rst`)
* [X] I have updated the appropriate documentation

### CODE REVIEW

File: SCons/Environment.xml [hunk 1]
```diff
@@ -883,16 +883,7 @@ disables derived file caching.
...
-<parameter>custom_class</parameter> which should be a class type which is a subclass of
-<classname>SCons.CacheDir.CacheDir</classname>, SCons will
-internally invoke this class to use for performing caching operations.
-This argument is optional and if left to default <constant>None</constant>, will use the
-default <classname>SCons.CacheDir.CacheDir</classname> class.
-</para>
-
-<para>
-Calling the environment method
+Calling the environment method form
```
Comment (bdbaddog) [inline SCons/Environment.xml]: form or from?

File: doc/user/caching.xml [hunk 2]
```diff
@@ -534,22 +513,25 @@ Program('prog', ['f1.c', 'f2.c', 'f3.c', 'f4.c', 'f5.c'])
...
-    You can then pass your custom class to the &f-link-CacheDir;
-    method or set the &consvar;
-    &cv-link-CACHEDIR_CLASS; to the class before configuring the cache
-    in that environment.
-    SCons will internally invoke and use your custom class when performing
+    To define custom cache behavior, subclass the
+    <classname>SCons.CacheDir.CacheDir</classname> class,
+    specializing those methods you want to change.
+    You can pass this custom class as the <parameter>custom_class</parameter>
+    paramter when calling &f-link-CacheDir; for global reach,
```
Comment (bdbaddog) [inline doc/user/caching.xml]: typo: paramter -> parameter

File: doc/user/caching.xml [hunk 3]
```diff
@@ -559,8 +541,8 @@ Program('prog', ['f1.c', 'f2.c', 'f3.c', 'f4.c', 'f5.c'])
 
     <scons_example name="custom_caching">
       <file name="SConstruct" printme="1">
-import SCons
 import os
+import SCons.CacheDir
```
Comment (bdbaddog) [inline doc/user/caching.xml:545]: Is this necessary and missing originall?

Comment (mwichmann) [reply doc/user/caching.xml:545]: Yes.  `CacheDir` the function is global to sconscripts, but not the class.
```py
scons: Reading SConscript files ...
NameError: name 'SCons' is not defined:
  File "/home/mats/github/scons/tmp/cdir/SConstruct", line 3:
    class CustomCacheDir(SCons.CacheDir.CacheDir):
```

File: doc/user/caching.xml (hunk 2 above)
Comment (mwichmann) [reply doc/user/caching.xml]: updated.  I seem to make that specific typo a lot.

**Pull request merged**